### DISCUSSION TITLE
Broken Images

### DISCUSSION THREAD

**Discussion by Moscow1991** [iptv-org/community#601] [Show and tell]
### Describe your issue

The icons of many Japanese IPTV channels cannot be displayed properly. It may be that their files located on imgur.com are damaged. I hope it can be repaired.

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by R4IN2007** [comment]
All imgur images are down and damaged. Might that the images are deleted?

**Comment by trincowski** [comment]
> All imgur images are down and damaged. Might that the images are deleted?

All? I don't have that problem...

**Comment by R4IN2007** [comment]
Some US channels images are down for example ABC4 Komo

![Screenshot_20240321_094217_Chrome](https://github.com/iptv-org/database/assets/83226646/38c34c3f-54cd-4f5a-9ffa-82a86a81f1fe)

**Comment by BellezaEmporium** [comment]
Might have been some sort of DMCA ? Weird enough to remove such image.

**Comment by Alstruit** [comment]
There are around ~1500 broken images. Not an easy fix

**Comment by Carlinhos027** [comment]
Sorry. Sometimes my finger is crazy lol

**Comment by Solitaire242** [comment]
> > All imgur images are down and damaged. Might that the images are deleted?
> 
> All? I don't have that problem...

I also have a problem with IPTV on air in Japan on PC and mobile.
